File: Go/uri_1071.go
package main

import (
	"fmt"
	"sort"
)

func main() {
	var v [2] int
	var soma = 0
	fmt.Scan(&v[0], &v[1])
	s := []int{v[0], v[1]}
	sort.Ints(s)
	for i := s[0] + 1; i < s[1]; i++ {
		if i%2 != 0 {
			soma += i
		}
	}
	fmt.Println(soma)
}

File: Python/uri_1071.py
v = []
x = 0
for i in range(0, 2):
    v.append(int(input()))
v.sort()
n = v[0] + 1
m = v[1]
for i in range(n, m):
    if i % 2 != 0:
        x += i
print(x)